### PR TITLE
GC Subway - Remove sub-items from the list in working example

### PR DESCRIPTION
**Pull request by GormFrank**
Also:
- got rid of the `.noline` class as requested by DTO in WET-147.
- added proper active menu item styles and semantic.

**Will probably conflict with #1751**

### CODE REVIEW

**Review by duboisp — APPROVED** [review]

**Comment by duboisp** [comment]
I tested @GormFrank changes and it works as expected. 

I am going to push additional changes + working example to show how to create mobile documentation navigation anchor.

**Comment by GormFrank** [comment]
Noticed that after adding the two `<a>` tags in the `<li>`s, the white circles have vanished in mobile view.

Also, the `.cnt-wdth-lmtd` class causes the bottom border to be cut short from the right end as you move from the desktop to the mobile view.

**Comment by GormFrank** [comment]
Noticed that after adding the two `<a>` tags in the `<li>`s, the white circles have vanished in mobile view.

Also, the `.cnt-wdth-lmtd` class causes the bottom border to be cut short from the right end as you move from the desktop to the mobile view.

**Comment by duboisp** [comment]
@GormFrank I fixed both issue.

**Comment by GormFrank** [comment]
> @GormFrank I fixed both issue.

I'll look at it tomorrow first thing in the morning!

File: src/plugins/gc-subway/page2-en.hbs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	],
+	"secondlevel": false,
+	"dateModified": "2020-12-07",
+	"share": "true"
+}
+---
+{{>alert-provisional-demos}}
+
+<nav class="provisional gc-subway">
+	<h1 id="gc-document-nav">Service initiation template</h1>
```
Comment (delisma) [inline src/plugins/gc-subway/page2-en.hbs:19]: Can we replace **Service initiation template** by **Subway navigation component**?

File: src/plugins/gc-subway/page2-fr.hbs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	],
+	"secondlevel": false,
+	"dateModified": "2020-12-07",
+	"share": "true"
+}
+---
+{{>alert-provisional-demos}}
+
+<nav class="provisional gc-subway">
+	<h1 id="gc-document-nav">Gabarit d'initiation de service</h1>
```
Comment (delisma) [inline src/plugins/gc-subway/page2-fr.hbs:19]: Can we replace **Gabarit d'initiation de service** by **Composante de navigation de style métro**?

**Review by delisma — COMMENTED** [review]
We wouldn't want to convey that this is a template. The example should stick to the component idea as much as possible.  We may publish a template once it's stable.

**Pull request merged**